src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+
+  }
+
+  &__star--0 :nth-child(-n + 4) {
```

Comment:
I don't this that increasing specificity is a must here, :nth-child(-n + 4) will work.

Reply:
With this code you are ready only for the case when the rating is 4 stars. I think you need to generate modifiers using scss loop. (If you need help, tag me)